docs/api/app.md

```diff
@@ -1531,3 +1566,11 @@ or Windows [WOW](https://en.wikipedia.org/wiki/Windows_on_Windows)).
 You can use this property to prompt users to download the arm64 version of
 your application when they are running the x64 version under Rosetta
 incorrectly.
+
+### `app.notificationsComServerCLSID` _Windows_
```

Comment:
Per last review comment:

> Can we provide a link for where how to generate a CLSID, something like https://www.guidgenerator.com/online-guid-generator.aspx with Uppercase + Brackets + Hyphens checked is a reasonable way to do it.

> Or just documenting it's an upper case uuid with curly braces

This documentation is still required

Reply:
https://learn.microsoft.com/en-us/windows/win32/api/rpcdce/nf-rpcdce-uuidfromstring

Reply:
See commit https://github.com/electron/electron/commit/88f173b1bf6a883f0de2c06c1eeb74942a641bb0